templates/tableau-single-server-ubuntu.template

```diff
@@ -230,47 +230,50 @@
     },
     "Mappings": {
         "AwsRegionToAMI": {
+            "AMI": {
+                "US1604HVM": "Ubuntu-Server-16.04-LTS-HVM"
```

Comment:
I'm assuming that this is a plain-vanilla Ubuntu AMI...

Reply:
yes :)

Reply:
Alright. Our systems will maintain it to the latest as long as we can test it out in an automated fashion.

Reply:
sounds good!